application-tour-ui/src/main/resources/TourCode/TourJS.xml

```diff
@@ -276,8 +274,6 @@ require(['jquery', 'xwiki-meta'], function ($, xm) {
       tour.init();
       if (forceStart) {
         tour.restart();
-      } else {
-        tour.start();
```

Comment:
So if ``forceStart == false`` we don't start the tour anymore? Why calling ``init()`` in this case then?